lib/cog_api/client.ex

```diff
@@ -46,6 +46,8 @@ defmodule CogApi.Client do
   @callback relay_group_delete(String.t, %Endpoint{}) :: {atom, [String.t]}
   @callback relay_group_add_relay(String.t, String.t, %Endpoint{}) :: {atom, %RelayGroup{}}
   @callback relay_group_remove_relay(String.t, String.t, %Endpoint{}) :: {atom, %RelayGroup{}}
+  @callback relay_group_add_bundle(String.t, String.t, %Endpoint{}) :: {atom, %RelayGroup{}}
```

Comment:
I was having some issues with testing that, so I'd rather hold off for now. I don't _think_ we're going need that in Flywheel.